latest source build failed with urdfdom

### Generated by Generative AI

_No response_

### Operating System:

```Linux tomoyafujita 5.15.0-139-generic #149~20.04.1-Ubuntu SMP Wed Apr 16 08:29:56 UTC 2025 x86_64 x86_64 x86_64 GNU/Linux```

### ROS version or commit hash:

rolling source build: https://github.com/ros2/ros2/commit/58faf9a091f9cd0bf24e93bb559b09133a75fafd

### RMW implementation (if applicable):

_No response_

### RMW Configuration (if applicable):

_No response_

### Client library (if applicable):

_No response_

### 'ros2 doctor --report' output

<details><summary>ros2 doctor --report</summary>

```console
<COPY OUTPUT HERE>
```
</details>


### Steps to reproduce issue

https://docs.ros.org/en/rolling/Installation/Alternatives/Ubuntu-Development-Setup.html

### Expected behavior

entire source build should succeed.

### Actual behavior

fails with the following error.

```console
--- stderr: urdfdom
CMake Error at CMakeLists.txt:51 (find_package):
  Could not find a configuration file for package "urdfdom_headers" that is
  compatible with requested version "1.0".

  The following configuration files were considered but not accepted:

    /root/ros2_ws/colcon_ws/install/urdfdom_headers/lib/urdfdom_headers/cmake/urdfdom_headers-config.cmake, version: 2.0.0



---
Failed   <<< urdfdom [3.86s, exited with code 1]
```

### Additional information

_No response_

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,7 +48,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 find_package(tinyxml2_vendor QUIET)
 find_package(TinyXML2 REQUIRED)
 
-find_package(urdfdom_headers 1.0 REQUIRED)
+find_package(urdfdom_headers REQUIRED)
 find_package(console_bridge_vendor QUIET) # Provides console_bridge 0.4.0 on platforms without it.
 find_package(console_bridge REQUIRED)
 

